integreat_cms/static/src/js/media-management/library.tsx

```diff
@@ -177,57 +177,77 @@ const Library = ({
+                                const searchInput = document.getElementById("table-search-input") as HTMLInputElement;
+                                console.debug(`Search form submitted with query "${searchInput.value}"...`);
+                                if (!searchInput.value) {
+                                    console.debug(`Search query empty, returning to the home directory...`);
+                                    route("/");
+                                } else {
+                                    route(`/search/${encodeURIComponent(searchInput.value)}`);
+                                }
+                            }}>
+                            <input
+                                id="table-search-input"
+                                form="media-search-form"
+                                type="search"
+                                autocomplete="off"
+                                placeholder={mediaTranslations.btn_search}
+                                class="rounded-r-none"
+                                data-url={apiEndpoints.getSearchSuggestions}
+                                data-object-type="media"
+                                data-archived="false"
+                                value={searchQuery}
+                            />
+                            <div
+                                id="table-search-suggestions"
+                                class="absolute hidden shadow rounded-b top-full bg-graz-200 w-full z-10 max-h-60 overflow-y-auto cursor-pointer"
+                            />
+                        </form>
+                        <button
+                            id="search-submit-btn"
+                            title={mediaTranslations.btn_search}
+                            class="bg-blue-500 hover:bg-blue-600 text-white rounded-r py-2 px-3"
+                            type="submit"
+                            form="media-search-form">
+                            <Search className="w-5" />
+                        </button>
+                    </div>
+                    <div>
+                        <button
+                            id="unused-media-filter-btn"
+                            title={mediaTranslations.btn_filter_unused_media_files}
+                            className="btn"
+                            type="submit"
+                            action={apiEndpoints.filterUnusedMediaFiles}
+                            onClick={async () => {
+                                try {
+                                    route(`/filter-unused-media-files/`);
+                                } catch {
+                                    console.debug("Error");
+                                }
+                            }}>
+                            <Filter className="inline-block mr-2 h-5" />
+                            {mediaTranslations.btn_filter_unused_media_files}
+                        </button>
```

Comment:
> Maybe I'm doing something wrong, but I wasn't able to uncheck the filter and think it would be good, if it would also be possible to reset the filter after using it once. So far it is only possible when I remove the filter-unused-media-files from the URL

I agree with this - What about a second button here "Show all media files" which is hidden by default and only shown when the filter is applied? (similarly, the filter button could only be shown when it's not applied)